scripts/dev/licensetext.py

```diff
@@ -81,6 +81,11 @@ def previous():
     lines = lines[:-9]
     # Delete lines 4-6
     lines = lines[:4] + lines[7:]
+    # Modify the notice
+    lines[0] = lines[0].replace(' and', ',')
```

Comment:
I guess you didn't run this script. I don't see any changes regarding ' and' -> ','.

Reply:
This function takes the original text and modifies it up to the previous version, the function `current` modifies the original text up to where it is now.